main.go

```diff
@@ -9,33 +9,36 @@ import (
 	"github.com/kairos-io/AuroraBoot/deployer"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
+	"github.com/urfave/cli/v2"
+)
 
-	"github.com/urfave/cli"
+var (
+	// https://goreleaser.com/cookbooks/using-main.version/
+	version = "dev"
```

Comment:
in all other kairos artifacts we use a more especific version which also has the commit and go version. Maybe we should use the same here?
https://github.com/kairos-io/immucore/blob/main/internal/version/version.go

Reply:
That's what I went with initially but then I realized we don't use Earthly to build and release this one.I thought I should better follow the go-releaser convention. The alternative is to just pass a ldflags to goreleaser with whatever value we want. Should I change it?

Reply:
Actually, what you put in this `version` is up to the command that build the binary. With the changes in this PR, it's either passed as a build arg (when it's a tag) or we can pass it by hand when we build the image locally. Since we don't have an earthly target to build the binary, there is nothing else to implement, no?

Reply:
I pushed a commit to change this so that it uses `git describe` to calculate a version string. It still only applied to releases. The only difference is the image it builds when we push to master, which will now have a proper version like `v0.2.7-46-gd9f73da`.

Reply:
Checking that it works requires us to release of course. I suggest we do a patch release to test it.